app.json

```diff
@@ -38,6 +38,8 @@
     },
     "X_API_KEY": {
       "required": true
+    }, "DRS_URL": {
+      "required": true
```

Comment:
That is interesting! I didn't know about this file!

Reply:
Yeah, Ben W showed me this sneaky trick :)